src/Core/Compiler/CompilerService.cs

```diff
@@ -448,20 +470,27 @@ public IEnumerable<QsNamespaceElement> IdentifyElements(string source, ITaskRepo
                     return null;
                 }
 
+                // TODO before merging! Undo introducing generateCSharp param.
```

Comment:
(nit) It probably makes sense to extract this TODO to the description of the PR, with the checkbox.

Reply:
Resolved TODO.